src/storage/storage_structure/disk_overflow_file.cpp

```diff
@@ -11,20 +11,28 @@ using namespace kuzu::common;
 namespace kuzu {
 namespace storage {
 
+void DiskOverflowFile::unpinOverflowCache(OverflowCache& overflowCache) {
+    if (overflowCache.pageIdx != UINT32_MAX) {
+        bufferManager.unpin(*overflowCache.fileHandle, overflowCache.pageIdx);
+    }
+}
+
 void DiskOverflowFile::readStringsToVector(TransactionType trxType, ValueVector& valueVector) {
     assert(!valueVector.state->isFlat());
+    OverflowCache overflowCache;
```

Comment:
I'm slight concerned whether this object creation will cause overhead. Usually the compiler optimize these things so it shouldn't matter. 

But can u try to benchmark a bit to see if we become slower if we wrap things as a struct? If we do become slower I would use three primitive types instead.